script/helpers/Governors.sol

```diff
@@ -0,0 +1,266 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {
+  GovernorVotesComp,
+  ERC20VotesComp
+} from "openzeppelin-flexible-voting/governance/extensions/GovernorVotesComp.sol";
+import {GovernorVotes} from "openzeppelin-flexible-voting/governance/extensions/GovernorVotes.sol";
+import {GovernorTimelockControl} from
+  "openzeppelin-flexible-voting/governance/extensions/GovernorTimelockControl.sol";
+import {ERC20Votes} from "openzeppelin-flexible-voting/token/ERC20/extensions/ERC20Votes.sol";
+import {
+  Governor as FlexGovernor,
+  Governor,
+  IGovernor
+} from "openzeppelin-flexible-voting/governance/Governor.sol";
+import {TimelockController} from "openzeppelin-flexible-voting/governance/TimelockController.sol";
+import {GovernorTimelockCompound} from
+  "openzeppelin-flexible-voting/governance/extensions/GovernorTimelockCompound.sol";
+import {ICompoundTimelock} from "openzeppelin-flexible-voting/vendor/compound/ICompoundTimelock.sol";
+
+import {GovernorCountingFractional} from "flexible-voting/src/GovernorCountingFractional.sol";
+
+contract GovernorTestnetSettings {
+  function quorum(uint256) public view virtual returns (uint256) {
+    return 1_000_000;
```

Comment:
Shouldn't this be `1_000_000e18` if we want it to produce results that look like a "real" DAO with our Decimals 18 tokens?

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/82/commits/ea5da721be6dca05841003f7ca43335e1dd662ea